### PR TITLE
Add a new tank named swift tank

### PR DESCRIPTION
**Pull request by zq-chen22**
Just some nonsense in readme

### CODE REVIEW

**Comment by zq-chen22** [comment]
Add a new tank named by swift tank, which has three features.
Firstly, it rotates and moves faster.
Secondly, it have a "flesh": When you type in ‘F’ for the first time, you can flesh forward.
Thirdly, it will charge for a quintiple hit (五倍伤害的重击） every 6 seconds.

File: src/battle_game/core/CMakeLists.txt [hunk 1]
```diff
@@ -3,7 +3,7 @@ list(APPEND LIBRARY_LIST ${lib_name})
 
 add_library(${lib_name})
 file(GLOB_RECURSE source_files *.cpp *.h)
-target_sources(${lib_name} PRIVATE ${source_files})
+target_sources(${lib_name} PRIVATE ${source_files} "units/swift_tank.h" "units/swift_tank.cpp")
```
Comment (qinchuanhui) [inline src/battle_game/core/CMakeLists.txt:6]: Don't do this change

File: README.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 [![Linux Build](https://github.com/Yao-class-cpp-studio/battle_game/actions/workflows/linux-build.yml/badge.svg)](https://github.com/Yao-class-cpp-studio/battle_game/actions/workflows/linux-build.yml)
 
-经过一个学期的学习，我们已经一起学习了许多编程知识，大家的编程技能想必也有了显著的提高。
+经过一个学期的学习，我们已经一起学习了许多编程知识，大家的编程技能想必也有了显著的提高(doge)。
```
Comment (qinchuanhui) [inline README.md:5]: Undo this change

**Review by qinchuanhui — CHANGES_REQUESTED** [review]
Others LGTM

**Comment by zq-chen22** [comment]
Task 4 
1.help to make firebullet (done with 苏钰乐）
2.add a 'f' flesh (applied for swift tank)

**Review by qinchuanhui — APPROVED** [review]
LGTM, you can get the corresponding score

**Comment by qinchuanhui** [comment]
> Task 4 1.help to make firebullet (done with 苏钰乐） 2.add a 'f' flesh (applied for swift tank)

seems this task haven't be done in this PR

**Pull request closed without merging**